polars/polars-cli/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "polars-cli"
+version = "0.1.0"
```

Comment:
@ritchie46 since this is an independent application, I didn't think it should be versioned the same as the rest of them. 

Originally I thought of hoisting it up to the root (`/polars-cli`), as `py-polars` & `polars` are both independently versioned.

Reply:
Yes, maybe it is a good idea to hoist to the root. :+1: